src/util/Components/Forms/InputNumber.tsx

```diff
@@ -29,13 +47,14 @@ const InputNumber = (props: PropsType) => {
         {props.header}
       </span>
       <input
-        className={"form-control " + props.inputClass}
+        className={"form-control " + (props.inputClass || "")}
         type="number"
         placeholder="Max Rating"
-        value={props.value}
+        value={inputValue}
         name={props.name}
         step={props.step ? props.step : 1}
         onChange={(e) => {
+          setInputValue(e.target.value);
           let num: number = parseInt(e.target.value);
 
           num = processNumber(num, props.min, props.max);
```

Comment:
```suggestion
        onChange={(e) =>  setInputValue(e.target.value) }
```
Remaining codes are redundant. Rest of the things looks good to me. Nice work!